scripts/install_curl.sh

```diff
@@ -45,16 +45,39 @@ fi
 
 pushd ${SRCDIR}
 
+# TODO: Ignore HTTP3 socket connection failures
+pushd $1
+patch -p1 <<'EOF'
+diff --git a/lib/cf-socket.c b/lib/cf-socket.c
+index e42b4a87b..f99fcd80c 100644
+--- a/lib/cf-socket.c
++++ b/lib/cf-socket.c
+@@ -1650,7 +1650,7 @@ static CURLcode cf_udp_setup_quic(struct Curl_cfilter *cf,
+ 
+   rc = connect(ctx->sock, &ctx->addr.sa_addr, ctx->addr.addrlen);
+   if(-1 == rc) {
+-    return socket_connect_result(data, ctx->r_ip, SOCKERRNO);
++    /* return socket_connect_result(data, ctx->r_ip, SOCKERRNO); */
```

Comment:
This feels wrong - why are we disabling this?

Reply:
This is quite hacky as you noticed, so I've specially marked it with a TODO. The harness uses AF_UNIX sockets and it makes the connect() call fail, which blocks fuzzing. Ideally we would have something nicer than this patch on cURL itself, to allow for fuzzing to proceed.